clang/test/SemaSYCL/intel-max-work-group-size-device.cpp

```diff
@@ -71,13 +71,13 @@ int main() {
 
     // CHECK-LABEL: FunctionDecl {{.*}}test_kernel4
     // CHECK:       SYCLIntelMaxWorkGroupSizeAttr {{.*}}
+    // CHECK-NEXT:  UnaryOperator{{.*}} 'int' prefix '-'
     // CHECK-NEXT:  IntegerLiteral{{.*}}8{{$}}
     // CHECK-NEXT:  IntegerLiteral{{.*}}8{{$}}
-    // CHECK-NEXT:  UnaryOperator{{.*}} 'int' prefix '-'
     // CHECK-NEXT:  IntegerLiteral{{.*}}8{{$}}
-    // expected-warning@+2{{the resulting value of the 'max_work_group_size' attribute parameter 2 is always non-negative after implicit conversion}}
+    // expected-warning@+2{{the resulting value of the 'max_work_group_size' attribute first parameter is always non-negative after implicit conversion}}
```

Comment:
Should it be "third" parameter?

Reply:
value gets swapped in device mode, Please see the AST, so it will be first one.

Reply:
Yeah, but in code it looks like third parameter. Seems also a bit confusing.

Reply:
Yes, i agree, This is what @AaronBallman mentioned on https://github.com/intel/llvm/pull/2906#discussion_r546919084
It was decided to do it in a separate PR. This seems like a bug in original WorkGroup Attribute implementation, I have created #2938 for this.

Reply:
> @Fznamznon, What do you think? is it OK to do it in a separate PR?

Ok

Reply:
Thanks Mariya.